### PR TITLE
popup

### PR DESCRIPTION
**Pull request by Westerinoo**
#### HTML-CSS_PopUp

[Demo](https://westerinoo.github.io/HTML-CSS_PopUp/) |
[Code base](https://github.com/Westerinoo/HTML-CSS_PopUp)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/Westerinoo/HTML-CSS-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (A-Ostrovnyy) [inline submissions/Westerinoo/HTML-CSS-popup/index.html:7]: Add proper page title.

File: submissions/Westerinoo/HTML-CSS-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+           
+                <input id="checkbox__popup" type="checkbox" class="checkbox__popup">
+                <div class="wrap-popup">
+                <div class="popup__content">
+                        <ul class="popup__lists">
+                           <li class="popup__items"><a href="#"><img class="icons" src="./icons/calendar.png" alt="icon-calendar"><span>calendar</span></a></li>   
+                            <li class="popup__items"><a href="#"><img  src="./icons/mail.png" alt="icon-mail" class="icons"><span>mail</span></a></li> 
+                            <li class="popup__items"><a href="#"><img  src="./icons/news.png" alt="icon-news" class="icons"><span>news</span></a></li> 
+                            <li class="popup__items"><a href="#"><img  src="./icons/maps.png" alt="icon-maps" class="icons"><span>maps</span></a></li> 
+                            <li class="popup__items"><a href="#"><img  src="./icons/files.png" alt="icon-files" class="icons"><span>files</span></a></li>
```
Comment (A-Ostrovnyy) [inline submissions/Westerinoo/HTML-CSS-popup/index.html:26]: Use [Prettier](https://prettier.io/) for formatting code

File: submissions/Westerinoo/HTML-CSS-popup/style.css [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+}
+  
+a{
+    height: 100%;
+    display: grid;
+    justify-items: center;
+    align-items:center;
+   
+}
+a:focus{
```
Comment (A-Ostrovnyy) [inline submissions/Westerinoo/HTML-CSS-popup/style.css:113]: Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css). Avoid using too specific CSS selectors. [Read about it.](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!

**Review by Westerinoo — COMMENTED** [review]
corrected

File: submissions/Westerinoo/HTML-CSS-popup/index.html (hunk 2 above)
Comment (A-Ostrovnyy) [reply submissions/Westerinoo/HTML-CSS-popup/index.html]: Still not fixed :cry:

File: submissions/Westerinoo/HTML-CSS-popup/index.html [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+                            <li class="popup__items"><a class  ="link-popup" href="#"><img  src="./icons/search.png" alt="icon-search" class="icons"><span>search</span></a></li> 
+                            <li class="popup__items"><a class  ="link-popup" href="#"><img  src="./icons/photos.png" alt="icon-photo" class="icons"><span>photos</span></a></li> 
+                            <li class="popup__items"><a class  ="link-popup"href="#"><img  src="./icons/apps.png" alt="icon-apps" class="icons"><span>apps</span></a></li> 
+                            <li class="popup__items"><a class  ="link-popup" href="#"><img  src="./icons/documents.png" alt="icon-document" class="icons"><span>documents</span></a></li> 
+                        </ul> 
+                               <input type="checkbox" class="checkbox__popup-more" id="checkbox__popup-more">
+                                <label class="label__popup-more" for="checkbox__popup-more">
+                                       <span>More</span> 
+                                </label>
```
Comment (A-Ostrovnyy) [inline submissions/Westerinoo/HTML-CSS-popup/index.html:38]: Add `cursor: pointer` to indicate to the user about an interactive element.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good progress!
Please don't resolve the comments. Comments must be resolved by the author.
Also, fix your first message in PR according to [this instraction](https://github.com/kottans/frontend-2022-homeworks/pull/308#issuecomment-1229189200)

File: submissions/Westerinoo/HTML-CSS-popup/index.html [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+        <ul class="nav__lists">
+           <li class="nav__items"><a class="link" href="#">Mail</a></li>
+           <li class="nav__items"><a class="link" href="#">Images</a></li>
+           <li class="nav__items">
+           
+                <input id="checkbox__popup" type="checkbox" class="checkbox__popup">
+                <div class="wrap-popup">
+                <div class="popup__content"> 
+                        <ul class="popup__lists">
+                           <li class="popup__items"><a class   ="link-popup" href="#"><img class="icons" src="./icons/calendar.png" alt="icon-calendar"><span>calendar</span></a></li>
```
Comment (A-Ostrovnyy) [inline submissions/Westerinoo/HTML-CSS-popup/index.html:22]: For decorative images implemented with `img` tag `alt` attribute must be an empty value. Icons are decorative images. Please set an empty value to `alt` attribute.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
One last issue)

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!
For future work don't resolve comments. Comments must be resolved by the author.

**Pull request merged**